src/test/resources/README.md

```diff
@@ -0,0 +1,53 @@
+# Skript-parser Testing System
+All files that are part of the internal testing structure of skript-parser can be found in here.
+
+## Contributing to Tests
+We're not very strict about what can be added to our test suite.
+Tests don't have to be good code - often, testing edge cases pretty
+much requires weird code. That being said, there are a couple of things
+to keep in mind:
+
+* Use tabs for indentation
+* Always test each possible case of your syntax.
+* Write descriptive assert messages and write comments
+* Ensure your tests pass for the latest stable version and the latest beta version
+
+Furthermore, we want to keep consistency across all tests:
+* Syntax tests should be put in their respective folder 
+(expressions go into the `expressions/` folder, and so forth).
+    * Their name should be equal to their Java class. `ExprLength.java`'s test for example,
+    will be saved at the following path: `expressions/ExprLength.txt`.
```

Comment:
```suggestion
    will be saved at the following path: `expressions/ExprLength.sk`.
```

Reply:
Okay so I saw that you did all tests using ``.txt`` extension, but I don't understand why. It's Skript scripts, so it should be ``.sk`` files

Reply:
1. All scripts are currently .txt, since we never actually talked about this. 
2. We might and will probably choose a new name in the future which would mean we’d have to change them all over again.
3. The main parser supports all extensions, as long as the file itself is a text file. So I don’t see the point in making the change here. Besides, I never really understood the .sk extension: if it’s a text file, it’s a text file. Using another extension will only make things more complicated.

Reply:
> 2. We might and will probably choose a new name in the future which would mean we’d have to change them all over again.

find and replacxe